### PR TITLE
DTFS2-5078 : Update cover start date

### PR DESCRIPTION
**Pull request by abhi-markan**
* `Change` link appears post cover start confirmation by the maker

### CODE REVIEW

**Review by ttbarnes — COMMENTED** [review]
Tests 😝

File: gef-ui/server/utils/helpers.js [hunk 1]
```diff
@@ -159,6 +161,8 @@ const previewItemConditions = (previewParams) => {
      * add link displayed taking to unissued-facility-change change page
      */
     summaryItems = previewSummaryItems(unissuedHref, unissuedShow, item);
+  } else if (details.coverDateConfirmed && item.id === 'coverStartDate') {
```
[the remark this answers is not in the document]
Comment (abhi-markan) [reply gef-ui/server/utils/helpers.js:164]: `coverDateConfirmed` will only be set for MIA

File: gef-ui/server/utils/helpers.js [hunk 2]
```diff
@@ -159,6 +160,8 @@ const previewItemConditions = (previewParams) => {
      * add link displayed taking to unissued-facility-change change page
      */
     summaryItems = previewSummaryItems(unissuedHref, unissuedShow, item);
+  } else if (Boolean(app.ukefDecisionAccepted) && item.id === 'coverStartDate') {
```
Comment (abhi-markan) [inline gef-ui/server/utils/helpers.js:163]: `ukefDecisionAccepted` is only set to true for an MIA.

File: gef-ui/server/utils/helpers.js [hunk 3]
```diff
@@ -220,6 +223,8 @@ const summaryItemsConditions = (summaryItemsObj) => {
   const changedToIssueShow = id === 'name' || id === 'coverStartDate' || id === 'coverEndDate' || id === 'issueDate';
   // column key to display add if facility not yet issued
   const unissuedShow = id === 'hasBeenIssued';
+  // Issued facility change link (post confirmation)
+  const issuedHref = `/gef/application-details/${app._id}/${data.details._id}/confirm-cover-start-date`;
```
Comment (ttbarnes) [inline gef-ui/server/utils/helpers.js:227]: Would be good to add a unit test for this, otherwise LGTM

Comment (abhi-markan) [reply gef-ui/server/utils/helpers.js:227]: Added

**Pull request merged**